kerby-backend/ldap-backend/src/main/java/org/apache/kerby/kerberos/kdc/identitybackend/LdapIdentityBackend.java

```diff
@@ -51,13 +51,18 @@
 import java.util.Collections;
 import java.util.List;
 import java.util.Map;
+import java.util.HashMap;
 
 /**
  * An LDAP based backend implementation.
  */
 public class LdapIdentityBackend extends AbstractIdentityBackend {
+    private int currentConnectionIndex = 0;
+    private Map<String, LdapConnection> connections = new HashMap<>();
```

Comment:
Should this be ConcurrentHashMap for thread-safety?

Reply:
@coheigea  Thanks for your review.

The changes in `connections` is in the constructor of LdapIdentityBackend or `doInitialize`, which should not be called 
 multi-thread, so I think there is no thread-safety problem, anything I missed?